.devcontainer/docker-compose.yml

```diff
@@ -41,7 +41,14 @@ services:
       - bash
       - -c
       - |-
-          ./scripts/run_tests.sh
+          if [ -f .env ]; then
```

Comment:
what does this if statement do?

Reply:
oh I see, you're feeding the environment variable in so that `DEBUG` would be available within the tests